fog/test-client/src/config.rs

```diff
@@ -129,6 +129,10 @@ pub struct TestClientConfig {
     /// What token id to use for the test
     #[clap(long, env = "MC_TOKEN_ID", default_value = "0")]
     pub token_id: TokenId,
+
+    /// Additional token ids to use for the test
+    #[clap(long, env = "MC_EXTRA_TOKEN_IDS", use_value_delimiter = true)]
+    pub extra_token_ids: Vec<TokenId>,
```

Comment:
lmk if you guys think it's better to just have `token_ids`, the thing that's annoying is I can't set a default value for it then afaict.

but if we coordinate with ops then we can do a breaking change here i think, which would make the API simpler

Reply:
It would be better, but not a huge concern. If it has an ops dependency, something we should address as a separate concern.

Reply:
I think you can set a default by making it not required, and when you use it check if it's empty and if so default to whatever you want. Since there's already a `token_ids()` method it can be changed to have this behavior. I would just make the `token_ids` field in this struct not `pub` so this behavior is always enforced.

Reply:
you were right remoun, I had some misconceptions about how this works.